lib/apple_sign_in/config.rb

```diff
@@ -16,6 +16,7 @@ def config
   end
 
   class Config
-    attr_accessor :apple_aud, :apple_iss
+    attr_accessor :apple_client_id, :apple_private_key , :apple_key_id,
+                  :apple_team_id, :redirect_uri
```

Comment:
Maybe I would leave this proposal for the next iteration but I would make the Config accessors readonly and allow to change its values only through the method `AppleSignIn.config { |config| }` to track the points where the application changes a config value